app/controllers/books_controller.rb

```diff
@@ -28,7 +28,7 @@ def create
 
     respond_to do |format|
       if @book.save
-        format.html { redirect_to @book, notice: t('messages.create') }
+        format.html { redirect_to @book, notice: t('messages.create', model: t('activerecord.models.book')) }
```

Comment:
```suggestion
        format.html { redirect_to @book, notice: t('messages.create', model: Book.model_name.human) }
```

とした方がtypoしたときにすぐエラーになるのでベターです。

Reply:
Railsガイドを確認してみました。
モデル内のネストした翻訳へのアクセス方法など見直してみました。
ご指摘ありがとうございます。